### PR TITLE
Add missing documentation

### PR DESCRIPTION
**Pull request by squarefrog**
As discussed in https://github.com/KrauseFx/fastlane/issues/679, some of the actions documentation needs updating.

This Pull Request addresses that. :rocket:

### CODE REVIEW

**Comment by KrauseFx** [comment]
:+1:

**Comment by squarefrog** [comment]
This is now ready for review.

File: docs/Actions.md [hunk 1]
```diff
@@ -1063,6 +1073,20 @@ puts release['name']
 
 To get a list of all available values run `fastlane action get_github_release`.
 
+### import_from_git
```
Comment (KrauseFx) [inline docs/Actions.md]: We could also add the `import` command here, which just imports the `Fastfile` without any cloning.

File: docs/Actions.md [hunk 2]
```diff
@@ -1002,6 +1004,14 @@ add_git_tag(
 
 [Artsy](https://github.com/fastlane/examples/blob/master/Artsy/eidolon/Fastfile) uses `fastlane` to automatically commit the version bump, add a new git tag and push everything back to `master`.
 
+### git_pull
+
+Executes a simple `git pull` command
```
Comment (KrauseFx) [inline docs/Actions.md]: Executes a simple `git pull --tags` command

File: docs/Actions.md [hunk 3]
```diff
@@ -1002,6 +1004,14 @@ add_git_tag(
...
+
+```ruby
+git_pull # will run `git pull --tags`
```
Comment (KrauseFx) [inline docs/Actions.md]: So that we can remove the comment from here

**Comment by KrauseFx** [comment]
Looks great, thanks for working on this pull request. I added some comments :+1:

File: docs/Actions.md (hunk 1 above)
Comment (squarefrog) [inline docs/Actions.md]: Might have to place it somewhere else, as this is in the middle of a `git` section. How about right at the top?

**Comment by squarefrog** [comment]
I have added those changes for you.

**Comment by KrauseFx** [comment]
Perfect, thank you @squarefrog for your contribution :+1:

**Pull request merged**